tests/inputs/TestNode.cpp

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) 2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+#include "Node.h"
+#include "Tests.h"
+
+TEST(TestNode, base) {
+  dfl::inputs::Node node("0", 0.0);
+  dfl::inputs::Node node0("0", 10.);
+  dfl::inputs::Node node2("1", 4.5);
+
```

Comment:
Why the names node,node0 and then node2 ?

Reply:
node0 because it has the same node id as "node". node2 is an artifact of the draft if the test where there were node, node1n and node2